controllers/machine_controller_phases_test.go

```diff
@@ -603,7 +603,7 @@ func TestReconcileBootstrap(t *testing.T) {
 			Bootstrap: clusterv1.Bootstrap{
 				ConfigRef: &corev1.ObjectReference{
 					APIVersion: "bootstrap.cluster.x-k8s.io/v1alpha4",
```

Comment:
Not for this PR, but we should avoid to hardcode gkv like this, instead we should use `bootstrap.GroupVersion.String()` (so the GVK is defined in a single point)
Also, one nice thing of introducing builders, is that we will solve this problem once for all and simplify a lot those tests

Reply:
Yeah it's everywhere. But as we replace all those occurrences anyway I wouldn't refactor it now (as you wrote)

Reply:
I exported the group versions, I think this will be useful pretty soon.